src/home/NotificationCenter.test.tsx

```diff
@@ -0,0 +1,498 @@
+import { fireEvent, render } from '@testing-library/react-native'
+import * as React from 'react'
+import { Provider } from 'react-redux'
+import { openUrl } from 'src/app/actions'
+import { fetchAvailableRewards } from 'src/consumerIncentives/slice'
+import NotificationCenter from 'src/home/NotificationCenter'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { createMockStore, getElementText, getMockStackScreenProps } from 'test/utils'
+import {
+  mockCusdAddress,
+  mockE164Number,
+  mockE164NumberPepper,
+  mockPaymentRequests,
+} from 'test/values'
+
+const TWO_DAYS_MS = 2 * 24 * 60 * 1000
+const BACKUP_TIME = new Date().getTime() - TWO_DAYS_MS
+
+const testNotification = {
+  ctaUri: 'https://celo.org',
+  priority: 20,
+  content: {
+    en: {
+      body: 'Body Text',
+      cta: 'Start',
+      dismiss: 'Dismiss',
+    },
+  },
+}
+
+const storeDataNotificationsDisabled = {
+  account: {
+    backupCompleted: true,
+    dismissedInviteFriends: true,
+    dismissedGetVerified: true,
+    accountCreationTime: BACKUP_TIME,
+    celoEducationCompleted: true,
+    dismissedStartSupercharging: true,
+  },
+  escrow: {
+    sentEscrowedPayments: [],
+  },
+  paymentRequest: {
+    incomingPaymentRequests: [],
+    outgoingPaymentRequests: [],
+  },
+  home: {
+    notifications: {
+      testNotification: {
+        ...testNotification,
+        dismissed: true,
+      },
+    },
+  },
+}
+
+const testReward = {
+  amount: '2',
+  contractAddress: 'contractAddress',
+  createdAt: Date.now(),
+  index: 0,
+  proof: [],
+  tokenAddress: '0xcusd',
+}
+
+const superchargeSetUp = {
+  web3: {
+    account: 'account',
+  },
+  app: {
+    numberVerified: true,
+  },
+  supercharge: {
+    availableRewards: [testReward],
+  },
+}
+
+const superchargeWithoutRewardsSetUp = {
+  ...superchargeSetUp,
+  supercharge: {
+    availableRewards: [],
+  },
+}
+
+const mockcUsdBalance = {
+  [mockCusdAddress]: {
+    address: mockCusdAddress,
+    isCoreToken: true,
+    balance: '100',
+    symbol: 'cUSD',
+    usdPrice: '1',
+    priceFetchedAt: Date.now(),
+  },
+}
+
+const mockcUsdWithoutEnoughBalance = {
+  [mockCusdAddress]: {
+    address: mockCusdAddress,
+    isCoreToken: true,
+    balance: '5',
+    symbol: 'cUSD',
+    usdPrice: '1',
+    priceFetchedAt: Date.now(),
+  },
+}
+
+describe('NotificationCenter', () => {
+  it('renders empty state when there is no notifications at all', () => {
+    const store = createMockStore({ ...storeDataNotificationsDisabled })
+    const { queryByTestId, queryByText } = render(
+      <Provider store={store}>
+        <NotificationCenter {...getMockStackScreenProps(Screens.NotificationCenter)} />
+      </Provider>
+    )
+
+    expect(queryByTestId('NotificationCenter/EmptyState')).toBeTruthy()
+    expect(queryByText('youAreAllCaughtUp')).toBeTruthy()
```

Comment:
nit: use the `getBy*` methods when making positive assertions for better error messages https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence

Reply:
cool! thanks! 😄